pinc/CharacterSelector.inc

```diff
@@ -33,7 +33,7 @@ class CharacterSelector
             }
 
             foreach ($picker_set->get_subsets() as $code => $picker) {
-                $safe_code = bin2hex($code);
+                $safe_code = uniqid();
```

Comment:
if it just needs to be unique, why not do code + index or something. So if there are 2 with the same code, then you could append 1, 2, etc.? That would avoid any sleep issues.

Reply:
Using sertal number now. Interestingly putting "picker_{$pickerid++}" causes an error.